velox/docs/functions/spark/string.rst

```diff
@@ -150,6 +150,12 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
         SELECT substring('Spark SQL', -10, 3); -- "Sp"
         SELECT substring('Spark SQL', -20, 3); -- ""
 
+.. spark:function:: translate(string, matchStr, replaceStr) -> varchar
+    Returns a new translated string. It translates the character in ``string`` by a
```

Comment:
Would you generate the docs and verify that they look nice. Don't you need an empty line here?